odoo/addons/base/views/res_partner_views.xml

```diff
@@ -77,7 +77,7 @@
                 <form string="Contact">
                     <field name="is_company" invisible="1"/>
                     <field name="type" invisible="1"/>
-                    <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "image_128"}'/>
+                    <field name="image_1920" widget='image' class="oe_avatar" options='{"preview_image": "avatar_128"}'/>
```

Comment:
For all of those in all forms, we should at least add `<field name="avatar_128" invisible="1"/>` in them, so that if/when we fix the preview bug, at least we will already have the proper value. Without having the field in the view it will never recompute.